tailwind.config.js

```diff
@@ -1,6 +1,5 @@
 /** @type {import('tailwindcss').Config} */
 export default {
-  prefix: 'su-', 
```

Comment:
I removed the `su-` prefix from the config since as of [Decanter v7.1.0](https://github.com/SU-SWS/decanter/releases/tag/7.1.0) we no longer include the prefix out of box. Our documentation hasn't been updated yet to reflect this change 😅